S/SymEngine/build_tarballs.jl

```diff
@@ -37,9 +37,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    "GMP_jll",
-    "MPFR_jll",
-    "MPC_jll",
+    Dependency("GMP_jll"),
```

Comment:
Please, revert changes also in this file, we don't want to build a new version of SymEngine now

Reply:
My apologies, those changes slipped in by accident, I'll remove them ASAP

Reply:
Although I do think those packages should have a new release, see https://github.com/JuliaPackaging/Yggdrasil/issues/876 -- but of course not sneaked in via this PR (I was careless, again: sorry)

Reply:
@giordano fixed now (I hope)